pwnlib/pe/pe.py

```diff
@@ -0,0 +1,54 @@
+import pefile
+import os
+from pwnlib.log import getLogger
+log = getLogger(__name__)
+__all__ = ['PE']
+
+class PE(object):
+    """Wrapper around pefile.
+
+    Example:
+
+        .. code-block:: python
+
+           >>> calc = PE(which('calc.exe'))
```

Comment:
You'll have to add `docs/source/pe.rst` and add it to `index.rst` for this to show up in the docs and for the tests to get run.